### PR TITLE
Update links to new docs

### PR DESCRIPTION
**Pull request by sacha-l**

### CODE REVIEW

File: knowledge-map.md [hunk 1]
```diff
@@ -314,46 +310,36 @@ subjects you need to know to develop with Substrate. So here we just list out th
...
   It is about composing multiple pallets (can be understood as Rust modules) together in the runtime
   and hooking them up together.
 
-- [**Runtime Development: Execution**](https://docs.substrate.io/v3/concepts/execution),
-  this article describes how a block is produced, and how transactions are selected and executed
-  to reach the next "stage" in the blockchain.
-
-- [**Runtime Develpment: Pallets**](https://docs.substrate.io/v3/runtime/frame),
-  this article describes what the basic structure of a Substrate pallet is consists of.
+- **Transaction lifecycle**](https://docs.substrate.io/main-docs/fundamentals/transaction-lifecycle/): this article describes the lifecycle of a transaction in a Substrate blockchain.
```
Comment (NukeManDan) [inline knowledge-map.md:318]:
```suggestion
- [**Transaction lifecycle**](https://docs.substrate.io/main-docs/fundamentals/transaction-lifecycle/): this article describes the lifecycle of a transaction in a Substrate blockchain.
```

File: knowledge-map.md [hunk 2]
```diff
@@ -405,26 +388,26 @@ In Substrate, you can program smart contracts using [ink!](https://paritytech.gi
...
   This tutorial introduces how to perform and schedule a runtime upgrade as an on-chain
   transaction.
 
-- About [transaction weight and fee](https://docs.substrate.io/v3/runtime/weights-and-fees), and
-  [benchmarking your runtime to determine the proper transaction cost](https://docs.substrate.io/v3/runtime/benchmarking).
+- About [transaction weight and fee](https://docs.substrate.io/main-docs/build/tx-weights-fees/), and
+  [benchmarking your runtime to determine the proper transaction cost](https://docs.substrate.io/main-docs/test/benchmark/).
 
-- [Off-chain Features](https://docs.substrate.io/v3/concepts/off-chain-features)
+- [Off-chain operations](https://docs.substrate.io/main-docs/fundamentals/offchain-operations/)
```
Comment (NukeManDan) [inline knowledge-map.md:398]:
```suggestion
- [Offchain operations](https://docs.substrate.io/main-docs/fundamentals/offchain-operations/)
```

**Review by NukeManDan — APPROVED** [review]
Lgtm after a few nits 🙏🙌

File: labs/06-substrate-kitties.md [hunk 3]
```diff
@@ -12,14 +12,13 @@ Read through the task and submit your lab results as per the instructions under
 
 ## Task
 
-Complete parts I - II of [the Kitties tutorial](https://docs.substrate.io/tutorials/v3/kitties/pt1).
+Complete the [Kitties collectables workshop](https://sacha-l.github.io/substrate-collectables-workshop/).
```
[the remark this answers is not in the document]
Comment (sacha-l) [reply labs/06-substrate-kitties.md:15]: Will close as resolved, if you think otherwise please open another issue :)

**Pull request merged**